java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -456,4 +457,66 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Object[]> zrankWithScore(String key, String member);
+
+    // TODO add @link to ZMPOP when implemented
+    /**
+     * Blocks the connection until it pops one member-score pair from the first non-empty sorted set
+     * in the provided list of key names.<br>
+     * <code>BZMPOP</code> is the blocking variant of <code>ZMPOP</code>.<br>
+     * A member with the highest/lowest score is popped from the first sorted set that is non-empty,
+     * with the given <code>keys</code> being checked in the order that they are given.<br>
+     * To pop more than one element use {@link #bzmpop(double, String[], ScoreModifier, long)}.
+     *
+     * @see <a href="https://redis.io/commands/bzmpop/">redis.io</a> for more details.
+     * @apiNote <code>BZMPOP</code> is a client blocking command, see <a
```

Comment:
Did Andrew say to remove these?

Reply:
Will remove/update for all blocking commands (few of them already merged)
I copied this from python client - so we need to update it too in that case